coordinate/config.go

```diff
@@ -52,11 +56,14 @@ type Config struct {
 	// GravityRho is a tuning factor that sets how much gravity has an effect
 	// to try to re-center coordinates. See [2] for more details.
 	GravityRho float64
+
+	// metricLabels is the slice of labels to put on all emitted metrics
+	MetricLabels []metrics.Label
 }
 
 // DefaultConfig returns a Config that has some default values suitable for
 // basic testing of the algorithm, but not tuned to any particular type of cluster.
-func DefaultConfig() *Config {
+func DefaultConfig(metricLabels []metrics.Label) *Config {
 	return &Config{
```

Comment:
This would break backwards compatibility. Either add a new function or elect to post-creation edit the returned value to set the metrics field

Reply:
fixed